src/sentry/discover/models.py

```diff
@@ -40,10 +41,18 @@ class DiscoverSavedQuery(Model):
     date_updated = models.DateTimeField(auto_now=True)
     visits = BoundedBigIntegerField(null=True, default=1)
     last_visited = models.DateTimeField(null=True, default=timezone.now)
+    is_default = models.BooleanField(null=True, blank=True)
```

Comment:
nit: 🤔 actually should we call this something else? Maybe `is_homepage`?
```suggestion
    is_homepage = models.BooleanField(null=True, blank=True)
```

Reply:
Changed it to `is_homepage`! Thanks for the suggestion, `is_default` could become confusing